news/index.md

```diff
@@ -1,9 +1,50 @@
 # News
 
+## OCFL Editors Workshop at iPres
+
+**16-20 September 2024**
+The OCFL Editors met during the iPres conference to review to the use cases to be addressed in version 2.0 of the specification and come up with initial thoughts on how to proceed with each one. These thoughts have now been added to the [Confirmed: In Scope Issues](https://github.com/OCFL/Use-Cases/issues?q=is%3Aissue+is%3Aopen+label%3A%22Confirmed%3A+In-scope%22). The OCFL Editors now welcome further comments and feedback from the community before proceeding. 
+
+### The OCFL Editors
```

Comment:
Do we need to be repeating the "OCFL Editors" block for each post?

Reply:
I was following the pattern of previous postings. We could compress them into a single posting, and also include details on the proposed schedule as per the last Editors meeting.

Reply:
I think leaving it is fine, remember that technically these are press releases and we should be copying and pasting the whole press release and sending them to different email listservs.  not everyone knows who we are.  it also allows us to document who was on the editorial team at the time the press release was released.